### ISSUE TITLE
HTML Symbols not working with partials (.tml) files

### ISSUE THREAD

**Issue by cjzamora** [OSSPhilippines/temple#2]
### What version of Temple are you using?

0.1.3

### What OS are you experiencing this issue?

MacOS Ventura

### What Browser are you experiencing this issue?

Chrome

### Describe the Bug

Apparently HTML symbols are not working on partials `.tml` files. But is working with `.dtml` files.

### Expected Behavior

&copy; &nbsp; or any HTML symbols should work upon rendering.

### To Reproduce

1. Create a partial file `.tml` extension
2. Load the partial file on a `.dtml` extension ex: `index.dtml`
3. Add any HTML symbols ex: &copy; &nbsp;

**Comment by cblanquera** [comment]
In `page.dtml` I put the following code.

```html
<link rel="import" type="template" href="./footer-symbol.tml" />
<style>
  /* 2. Add tailwind directives */
  @tailwind base;
  @tailwind components;
  @tailwind utilities;
</style>
<script>
  import { env, props } from '@ossph/temple';
  const { BUILD_ID, APP_DATA, NODE_ENV } = env();
  const { title } = props();
</script>
<html>
  <head>
    <title>{title}</title>
    <link rel="stylesheet" type="text/css" href={`/build/${BUILD_ID}.css`} />
    <script data-app={APP_DATA} src={`/build/${BUILD_ID}.js`}></script>
    <if true={NODE_ENV !== 'production'}>
      <script src="/dev.js"></script>
    </if>
  </head>
  <body>
    <h1 class="text-center">© {title}</h1>
    <footer-symbol />
  </body>
</html>
```

and in `footer-symbol.tml` I put the following code.

```html
<footer>
  © 2021 <a href="https://www.ryotahori.com">Ryota Hori</a>
</footer>
```

and it seems to load symbols correctly.

![image](https://github.com/user-attachments/assets/f8015c51-3b25-491f-b6e4-865dbc8b7082)

Can you provide the code using symbols that produces your error?

**Comment by cjzamora** [comment]
This is how i'm doing the html symbols i'm using the entity/html code/hex code. Pasting exactly the symbols works. 

```footer.tml```
```
<div>
    <span>&copy; &#169; &#xa9; 2024 Footer</span>
</div>
```
<img width="263" alt="Screenshot 2024-09-05 at 12 38 25 PM" src="https://github.com/user-attachments/assets/858012c8-5e81-433f-a575-a703f63a4cda">

But... Putting the same html code on `index.dtml` file works.

```index.dtml```
```
 <body class="light">
    <header></header>
    <footer></footer>
    &copy; &#169; &#xa9; This is on the index.dtml file
  </body>
```
<img width="265" alt="Screenshot 2024-09-05 at 12 41 11 PM" src="https://github.com/user-attachments/assets/831a8797-29da-4938-9f97-6c7a11b75fdd">

**Comment by cblanquera** [comment]
Ahh yea I was able to re-create it thanks. This is the doing of the native `document.createText()`. I mapped how it works on the server. I'll try to disable this escape to see if it breaks anything and get back to you.

**Comment by cblanquera** [comment]
https://stackoverflow.com/questions/20941956/how-to-insert-html-entities-with-createtextnode

![image](https://github.com/user-attachments/assets/d8f535cc-290b-4431-a8cc-61becb079eb6)

I tried this and works

![image](https://github.com/user-attachments/assets/b1399c8e-7199-4b5b-99c1-a9223cba8eb4)
![image](https://github.com/user-attachments/assets/8e23822b-f1d4-466b-9f88-e1f4e73b9810)

**Comment by cblanquera** [comment]
You can also pass HTML entities from the parent 
![image](https://github.com/user-attachments/assets/51ecf173-d92d-446b-9fd3-3e93e3aed26f)
because this is the same as setting `innerHTML`.

**Comment by cblanquera** [comment]
Now fixed in 0.1.4 - https://github.com/OSSPhilippines/temple/releases/tag/0.1.4

**Comment by cblanquera** [comment]
closing due to stale.
